packages/jellyfish-api-core/__tests__/category/account/sendTokensToAddress.test.ts

```diff
@@ -0,0 +1,134 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { SelectionModeType, SendTokensOptions } from '../../../src/category/account'
+
+describe('SendTokenToAddress', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await setup()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  let addressA: string
+  let addressB: string
+  let addressC: string
+  let tokenAddress: string
+
+  async function setup (): Promise<void> {
+    tokenAddress = await container.call('getnewaddress')
+    addressA = await container.call('getnewaddress')
+    addressB = await container.call('getnewaddress')
+    addressC = await container.call('getnewaddress')
+
+    await createToken(tokenAddress, 'CAT', 100)
+    await createToken(tokenAddress, 'ETH', 100)
+    await createToken(tokenAddress, 'DETH', 100)
+
+    await accountToAccount('CAT')
+    await accountToAccount('ETH')
+    await accountToAccount('DETH')
```

Comment:
how to test with diff coins set up??
acc1 = acc2acc('bat' , 100)
acc2 = acc2acc('cat' , 60)
acc3 = acc2acc('dog', 80)

sendTokensToAddress((), [addr]: [2@cat], crumbs) // <-- its 100% select acc2, what's the point??

Reply:
```
await accountToAccount('CAT')
await accountToAccount('ETH')
await accountToAccount('DETH')
```
The reason for creating multiple tokens is to avoid side effects of other tests. So we can run each test with a different token to avoid side effect (balance getting reduced which will make it hard to assert in other test). 

how to test with diff coins set up??
acc1 = acc2acc('bat' , 100)
acc2 = acc2acc('cat' , 60)
acc3 = acc2acc('dog', 80)

In regards to the comment above, whenever the `accountToAccount` function is called, it does an account to account transaction to 3 accounts, `addressA, addressB, addressC` with different amount. 

```
   await container.call('accounttoaccount', [tokenAddress, { [addressA]: `${20}@${symbol}` }])
    await container.call('accounttoaccount', [tokenAddress, { [addressB]: `${30}@${symbol}` }])
    await container.call('accounttoaccount', [tokenAddress, { [addressC]: `${50}@${symbol}` }])

```
So `sendTokensToAddress((), [addr]: [2@cat], crumbs)` will be addressA because it has the lowest amount of `CAT`

Reply:
you can set up diff coins on each test, but we only need 1 coin to test selectionMode.
else can you confirm this?
```
// 3 diff coins set up
acc1 = acc2acc('bat' , 100)
acc2 = acc2acc('cat' , 60)
acc3 = acc2acc('dog', 80)

acc2acc(null, { [addrA]: 20@bat }, { crumbs })
which account will be deducted amount?? acc2 or acc1??
```

Reply:
@canonbrother , it will be deducted from acc2 of course. I wrote a test checking that.

Reply:
can you point me to the test??

Reply:
No, only test same token gets deducted. Please refer to this. https://github.com/DeFiCh/jellyfish/pull/323/files#r651480837